rendermodules/utilities/pillow_utils.py

```diff
@@ -1,3 +1,13 @@
 from PIL import Image
 
 Image.MAX_IMAGE_PIXELS = None
+
+# imports needed for rough_align_qc
+import os
+import matplotlib
+if os.environ.get('DISPLAY', '') == '':
+    matplotlib.use('Agg')
+
+import matplotlib.pyplot as plt, mpld3
```

Comment:
I'm not too familiar with it, but it's not clear to me what mpld3 is doing as a dependency, as I don't see where else it is called.
In any case, PEP8 would suggest to use multiple lines for this.
```suggestion
import matplotlib.pyplot as plt
import mpld3
```

Reply:
accepted.